app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,9 @@ h1,h2,h3,h4 {
 .text-secondary-dark {
     color: $dark-secondary-color !important;
 }
+.text-white {
+    color: $bg-gray !important;
```

Comment:
Usually CSS/SCSS linters encourage the use of 1 tab (2 spaces) for styles, and I do too, it makes the CSS styles more readable. Just FYI

Reply:
You should pretty much always avoid the use of `!important`, first try out all the other CSS priorities: `inline-styles > html tags > ids > classes`.

If you are using it to override library styles (like Bootstrap or Bulma) please refer to the library documentation for options on how to customize those styles, I'm pretty sure you are not going to find an `!important` suggestion in there.

This feature is very powerful and very hard to override, it will cause you problems once your style code base begins to grow, so I encourage you A LOT to look for alternatives in this or you next projects. If you need help, feel free to reach out! 😄

Reply:
Ok thank you so much, I will look at the bootstrap documentation to know how to override styles :D